### ISSUE TITLE
Using spawn in multiprocessing causes an error

### ISSUE THREAD

**Issue by sw005320** [espnet/espnet#1266]
After #1251 is merged, we encountered the following error.
```
# lm_train.py --config conf/lm.yaml --ngpu 1 --backend pytorch --verbose 1 --outdir exp/train_rnnlm_pytorch_lm_word100 --tensorboard-dir tensorboard/train_rnnlm_pytorch_lm_word100 --train-label data/local/wordlm_train/train.txt --valid-label data/local/wordlm_train/valid.txt --test-label data/local/wordlm_train/test.txt --resume --dict data/local/wordlm_train/wordlist_100.txt 
# Started at Sat Oct 12 15:09:23 UTC 2019
#
Traceback (most recent call last):
  File "/content/espnet/egs/an4/asr1/../../../espnet/bin/lm_train.py", line 170, in <module>
    mp.set_start_method('spawn')
  File "/usr/lib/python3.6/multiprocessing/context.py", line 242, in set_start_method
    raise RuntimeError('context has already been set')
RuntimeError: context has already been set
# Accounting: time=1 threads=1
# Ended (code 1) at Sat Oct 12 15:09:24 UTC 2019, elapsed time 1 seconds
```
@kan-bayashi, can you fix it?
@arendu told me that `mp.set_start_method('spawn', force=True)` may solve it.

**Comment by kan-bayashi** [comment]
Sure.

**Comment by kan-bayashi** [comment]
I made PR to fix this issue #1267.
